py/desiutil/modules.py

```diff
@@ -194,7 +194,10 @@ def configure_module(product, version, product_root, working_dir=None, dev=False
         module_keywords['needs_ld_lib'] = ''
     if isdir(join(working_dir, 'pro')):
         module_keywords['needs_idl'] = ''
-    if (exists(join(working_dir, 'setup.py')) and isdir(join(working_dir, product))):
+    if (exists(join(working_dir, 'setup.py')) and
+        (isdir(join(working_dir, product)) or
+         isdir(join(working_dir, product.lower())))
```

Comment:
As I mentioned in #172, these are not the only possible combinations.  This change also needs to be applied to the case where the package lives in a `py/` directory.  Finally, we need unit tests for all of these cases.